app/cafe.py

```diff
@@ -0,0 +1,17 @@
+import datetime
+
+import app.errors
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor.keys():
```

Comment:
The use of `visitor.keys()` is unnecessary here. You can simply use `if 'vaccine' not in visitor:` to check for the presence of the key.